### PR TITLE
Fix Sequel save.

### PR DESCRIPTION
**Pull request by JFernandezWM**
Fix Sequel save.

needed-by: [#126](https://github.com/westernmilling/contracto/issues/126)

### CODE REVIEW

File: spec/lib/synchronized_model/message_receive_spec.rb [hunk 1]
```diff
@@ -83,13 +86,13 @@ def updated_at=(_value); nil; end
...
-        let(:updated_at) { Time.now.to_i + 100 }
+        let(:updated_at_was) { Time.now }
+        let(:updated_at) { (Time.now + 1) }
 
-        it 'calls save!' do
+        it 'calls save' do
           subject
 
-          expect(mock_model_sequel).to have_received(:save!)
+          expect(mock_model_sequel).to have_received(:save)
```
Comment (cracell) [inline spec/lib/synchronized_model/message_receive_spec.rb:100]: Shouldn't we test for both? One test for the active record scenario and one for the sequel scenario?

Comment (cracell) [reply spec/lib/synchronized_model/message_receive_spec.rb:100]: Nevermind, didn't see the context of the test.

Comment (JFernandezWM) [reply spec/lib/synchronized_model/message_receive_spec.rb:100]: Added test.

File: lib/synchronized_model/message_receive.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'pry'
```
Comment (cracell) [inline lib/synchronized_model/message_receive.rb]: Remove me?

**Comment by cracell** [comment]
You added `synchronized_model-0.3.4.gem`? We don't save the packaged gemfile in the repo.

**Review by cracell — APPROVED** [review]

**Pull request merged**